core/src/main/java/org/jruby/RubyRational.java

```diff
@@ -590,6 +578,12 @@ public IRubyObject op_div(ThreadContext context, IRubyObject other) {
     /** nurat_fdiv
      * 
      */
+//    @Override
+//    @JRubyMethod(name = "fdiv")
+//    public IRubyObject fdiv(ThreadContext context, IRubyObject other) {
+//        return f_div(context, f_to_f(context, this), other);
+//    }
+
```

Comment:
I fixed some of these names on the 2.4 branch, so there may be some minor conflicts. No need to keep this commented code in any case.